packages/flutter_tools/templates/app/web/manifest.json.tmpl

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "{{projectName}}",
+    "short_name": "{{projectName}}",
+    "start_url": ".",
+    "display": "minimal-ui",
+    "background_color": "#0175C2",
+    "theme_color": "#0175C2",
+    "description": "{{description}}",
+    "orientation": "portrait-primary",
+    "prefer_related_applications": false,
+    "icons": [
+        {
+            "src": "icons/Icon-192.png",
```

Comment:
Specifically, we need a 192 and 512 Icon to meet the add to home screen requirements in https://developers.google.com/web/fundamentals/app-install-banners#criteria